docs/release.md

```diff
@@ -6,7 +6,7 @@
   - `git checkout master`
   - `git pull`
 2. Set the release environment variable, which should be incremented from the previous released version.
-  - `export RELEASE=3.8`
+  - `export COLLECTOR_RELEASE=3.8`
```

Comment:
I have the RELEASE environment variable set from when I did the upstream release and didn't want to accidentally set the collector release to the stackrox release.